src/test/enginesynctest.cpp

```diff
@@ -1937,29 +1964,18 @@ TEST_F(EngineSyncTest, HalfDoubleThenPlay) {
     EXPECT_DOUBLE_EQ(1.0,
             ControlObject::getControl(ConfigKey(m_sGroup2, "rate_ratio"))
                     ->get());
-
-    ProcessBuffer();
-
     EXPECT_NEAR(
             m_pChannel1->getEngineBuffer()->m_pSyncControl->getBeatDistance(),
             m_pChannel2->getEngineBuffer()->m_pSyncControl->getBeatDistance(),
             kMaxFloatingPointErrorHighPrecision);
 
-    ProcessBuffer();
-    ProcessBuffer();
-
     EXPECT_DOUBLE_EQ(87.5,
             ControlObject::getControl(ConfigKey(m_sGroup1, "bpm"))->get());
 
     EXPECT_NEAR(
             m_pChannel1->getEngineBuffer()->m_pSyncControl->getBeatDistance(),
             m_pChannel2->getEngineBuffer()->m_pSyncControl->getBeatDistance(),
             kMaxFloatingPointErrorHighPrecision);
-
```

Comment:
by removing these ProcessBuffers the two checks at the bottom are identical -- does it fail if play advances?

Reply:
Ah Ok, I did not realize that this was a test. I will revert it.